# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/moz.build
# Commit: c5dac46ef11d
# Full Hash: c5dac46ef11d44762d5eb61610e58f4d52aec88d
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - PA: Delete GPUDecoderModule wrapper. r=mattwoodrow
#   
#   Moved support logic into WMFDecoderModule.
#   
#   Depends on D54882
# ==============================================================================

diff -r fca05e774f88 -r c5dac46ef11d dom/media/ipc/moz.build
--- a/dom/media/ipc/moz.build	Tue Oct 20 23:26:33 2020 +0000
+++ b/dom/media/ipc/moz.build	Tue Oct 20 23:26:38 2020 +0000
@@ -13,7 +13,6 @@
 ]
 
 EXPORTS.mozilla += [
-    'GpuDecoderModule.h',
     'IRemoteDecoderChild.h',
     'RDDChild.h',
     'RDDParent.h',
@@ -36,7 +35,6 @@
 ]
 
 SOURCES += [
-    'GpuDecoderModule.cpp',
     'RDDChild.cpp',
     'RDDParent.cpp',
     'RDDProcessHost.cpp',